service/src/main/scala/org/apache/celeborn/server/common/service/config/DynamicConfig.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.celeborn.server.common.service.config;
+
+import org.apache.celeborn.common.internal.config.ConfigEntry;
+import org.apache.celeborn.common.util.Utils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Dynamic configuration is a type of configuration that can be changed at runtime as needed. It can be used at system level/tenant level.
+ * When applying dynamic configuration, the priority order is as follows: tenant level overrides system level,
+ * which in turn overrides static configuration(CelebornConf). This means that if a configuration is defined at the tenant level,
+ * it will be used instead of the system level or static configuration(CelebornConf). If the tenant-level configuration is missing,
```

Comment:
```suggestion
 * it will be used instead of the system-level or static configuration({@link CelebornConf}). If the tenant-level configuration is missing,
```